contracts/interchainTokenService/InterchainTokenService.sol

```diff
@@ -381,6 +389,8 @@ contract InterchainTokenService is IInterchainTokenService, AxelarExecutable, Up
         bytes calldata destinationAddress,
         uint256 amount
     ) external payable onlyTokenManager(tokenId) notPaused {
+        // TODO: what's the point of sendHash? block.number can't always enforce uniqueness
```

Comment:
It is to make tracking a bit easier and add the possibility to express two different but similar calls that arrive close to each other (as long as they are at least a block appart)

Reply:
i'm thinking that we can use commandId for it instead